spec/system/resolving_a_disputed_order_spec.rb

```diff
@@ -29,7 +29,9 @@
     allow_any_instance_of(OrderDetailNoticePresenter).to receive(:global_admin_must_resolve?).and_return(true)
     login_as logged_in_user
     visit facility_disputed_orders_path facility
-    click_link order_detail.id.to_s
+    find_link(
+      href: manage_facility_order_order_detail_path(facility, order, order_detail)
+    ).click
```

Comment:
I think this can also be written as:

```suggestion
    click_link(
      order_detail.id.to_s,
      href: manage_facility_order_order_detail_path(facility, order, order_detail)
    )
```